code/src/components/CheckboxForm.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+const CheckboxForm = ({input, setInput}) => {
+    return (
+        <div className="checkbox-form-div">
```

Comment:
It doesn't seem like you use `className="checkbox-form-div"` in your stylesheet. Do you think it's better if you remove it, if you are not using it?